semantic-conventions/src/tests/data/markdown/scope/scope.yaml

```
@@ -0,0 +1,12 @@
+groups:
+  - id: scope
+    prefix: scope
+    brief: >
```

```suggestion
    type: scope
    brief: >
```
Same as https://github.com/open-telemetry/build-tools/pull/114#discussion_r935327622.